app/models/notification.rb

```diff
@@ -32,7 +32,12 @@ def send_new_follower_notification(follow, is_read: false)
       return if follow.followable_type == "User" && UserBlock.blocking?(follow.followable_id, follow.follower_id)
 
       follow_data = Notifications::NewFollower::FollowData.coerce(follow).to_h
-      Notifications::NewFollowerWorker.perform_async(follow_data, is_read)
+      follower = User.find_by(id: follow.follower_id)
```

Comment:
If we're quering the db instead of taking `follow.follower_id` we kind of assume that it can be missing (or in general it can potentially be missing), so I suppose it would worth `return` in case there is no `follower`.